extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -121,10 +121,17 @@ export class AmpSidebar extends AMP.BaseElement {
       }
     });
 
+    const ariaLabel  = this.element.getAttribute('data-close-button-aria-label')
```

Comment:
I'd recommend:
```
const ariaLabel = this.element.getAttribute('data-close-button-aria-label') || 'Close the sidebar';
```

for consistency with other places that do this and also now you can remove the if/else below.